jpata/hepaccelerate-cms
# Update software

- use latest hepaccelerate (added more unit tests)
- make sure singularity recipe is provided
- add GPU kernel for Pisa DNN evaluation
- validate HPC scripts again
- include ROOT in automatic travis test
